src/simulation/openables/NestBoxEmpty.ts

```diff
@@ -0,0 +1,12 @@
+import LootTable from '../../structures/LootTable';
+import SimpleOpenable from '../../structures/SimpleOpenable';
+
+// itemID 5075 = Bird nest empty
+const NestBoxEmptyTable = new LootTable().every(5075);
```

Comment:
just make a constant in constants.ts with `const EMPTY_BIRD_NEST_ID = 5075;` and use that in these files